lib/src/models/view/calendar_editor.dart

```diff
@@ -3,7 +3,10 @@ import "dart:async";
 import "package:flutter/foundation.dart" show ChangeNotifier;
 
 import "package:ramaz/data.dart";
+import "package:ramaz/models.dart";
+import "package:ramaz/pages.dart";
 import "package:ramaz/services.dart";
+import "package:ramaz/src/models/view/schedule.dart";
```

Comment:
```suggestion

import "schedule.dart";
```

Since it's right next to this file, just import it regularly. If you don't end up using it though, be sure to delete it.